README.md

```diff
@@ -0,0 +1,18 @@
+# OpenTelemetry Operations Collector
+
+This repository is focused on building and packaging the OpenTelemetry Collector with the necessary tools in different OSs to be compatible with Google Cloud Monitoring.
+
+## Prerequisite
+
+[docker](https://docs.docker.com/engine/install/) is installed
+
+## Building the Docker Image
+
+Run `make docker-build-image`
+
+## Collector for Linux
```

Comment:
Using the docker image to run these make commands is useful if you want to replicate what the CI pipeline is doing directly, but most of the time users would just want to run the make commands directly imo. I'd probably have the initial example here call `make build-tarball`

Then add a brief paragraph at the end saying "alternatively you can run the make commands inside docker. First, generate a docker image for running the build commands by running `make docker-build-image` and then run `make TARGET=build-tarball docker-run`"